src/main/java/com/example/TravelAgency/user/UserDTO.java

```diff
@@ -0,0 +1,25 @@
+package com.example.TravelAgency.user;
+
+import lombok.Value;
+
+
+@Value
+public class UserDTO {
+
+   // Long id;
```

Comment:
Id będzie potrzebne jak będziesz chciała wykonać jakieś zmiany w obiekcie w bazie.
Jeżeli zmapujesz obiekt który przyszedł z Resta, nie przepisując idka to zostanie zapisany nowy obiekt a nie zmodyfikowany już istniejący